File: python/백준/구현/1475 방 번호.py
nums = input().strip()

cnt = [0] * 10
for ch in nums:
    cnt[int(ch)] += 1

# 6과 9는 서로 호환 → 합친 뒤 올림 나눗셈
cnt[6] = cnt[9] = (cnt[6] + cnt[9] + 1) // 2

print(max(cnt))

File: cpp/백준/구현/1475 방 번호.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(nullptr);

    string nums;
    cin >> nums;

    vector<int> cnt(10, 0);
    for (char ch : nums) {
        cnt[ch - '0']++;
    }

    cnt[6] = cnt[9] = (cnt[6] + cnt[9] + 1) / 2;

    cout << *max_element(cnt.begin(), cnt.end()) << "\n";
    return 0;
}
